app/src/main/java/com/sdp/movemeet/view/activity/ActivityDescriptionActivity.java

```diff
@@ -63,12 +63,7 @@
     @VisibleForTesting(otherwise=VisibleForTesting.PRIVATE)
     public static boolean enableNav = true;
 
-    DrawerLayout drawerLayout;
```

Comment:
Yes, in each of our activities, we don't need those DrawerLayout-related lines, thanks for removing them!

Reply:
Yeah, it's cleaner !